releasenotes/notes/kolla-kayobe-version-check-f44d43c9c34d1b89.yaml

```
@@ -0,0 +1,12 @@
+---
+features:
+  - |
+    Ansible playbooks to check the installed Kayobe/Kolla-Ansible versions
+    with the expected versions in Kayobe configuration. These checks
+    will run on Kayobe bootstrap, host and service operations.
+upgrade:
+  - |
+    Checks are enabled by default which may break existing deployments using
+    custom forks or branches for Kayobe and Kolla-Ansible, to disable version
+    checks in configuration set ``stackhpc_enable_kayobe_check`` and
+    ``stackhpc_enable_kolla_ansible_check`` to false
```

nit: wording could be improved
```suggestion
upgrade:
  - |
    Kolla Ansible and Kayobe version checks are enabled by default which
    may fail on existing deployments using custom forks or branches for
    Kayobe and Kolla-Ansible. To disable version checks in configuration
    set ``stackhpc_enable_kayobe_check`` and
    ``stackhpc_enable_kolla_ansible_check`` to false.
```